GAME_DESIGN_PRINCIPLES.md

```diff
@@ -62,14 +57,17 @@ For example, in the "Pre-Caves Age," the player will not have access to coal, as
 - Find an above-ground structure that can grant the player coal.
 - Dig straight down until you find coal.
 
-Of course, this problem is completely solved as soon as the player gets coal, as they can now explore caves to find more coal with their newfound torches, but it's important that the player has these options in the first place to prevent them from getting stuck on a seemingly insurmountable barrier. This also gives them more freedom in how they can approach the game. Not one option is better or easier than the other, and if one turns out to be, then it should be allowed to be adjusted to encourage other methods.
+This problem is completely solved as soon as the player gets coal, as they can now explore caves to find more coal with their newfound torches. It's important that the player has these options to prevent them from getting stuck on a seemingly insurmountable barrier. This gives them more freedom in how they can approach the game, as no one option is particularly the best and can change depending on situation and location.
 
 ### Explain when Needed
-If something progress-related has no obvious explanation, the player will have to rely on a wiki to find out how to progress. One example is the bellows; players won't know to place it next to a furnace and jump on it repeatedly. The issue can be fixed by adding a tooltip that tells them how it works, while leaving out details they can intuitively find out themselves.
+Progress-related mechanics need explanations so the players don't get stuck. Keep it simple on the surface so it's easily approachable, but leave out details that the player can find themselves through intuition.
 
-Having a mechanic that isn't explained or illuded to makes it an "Invisible Mechanic." These can only be found out through looking at the game's code, wiki reading, or asking a developer, thus giving a disadvantage to casual players. It's best to avoid these at all costs.
+### Show, Don't Tell
+An important rule, showing how a mechanic works instead of telling encourages player experimentation and reduces confusion. Here are some examples:
```

Comment:
I don't think we can really do this. Some things cannot be discovered at all (e.g. gem encasing in tool crafting) and some things are better told up front, we don't want the player to die from food poisoning, just because no one warned them. This is the kind of stuff that makes you check the wiki before eating new food types.